README.md

```diff
@@ -149,6 +149,7 @@ Table of Contents
   * [gitea.com](https://www.gitea.com/) - Unlimited public and private Git repos
   * [github.com](https://github.com/) — Unlimited public repositories and unlimited private repositories (with unlimited collaborators) with Pipelines for CI/CD
   * [gitlab.com](https://about.gitlab.com/) — Unlimited public and private Git repos with unlimited collaborators, with Pipelines for CI/CD, and 10 GB space per repo
+  * [heptapod.net](https://heptapod.net/) — Heptapod is a friendly fork of GitLab Community Edition providing support for Mercurial
```

Comment:
Maybe better if the link went to https://foss.heptapod.net/

Reply:
Yeah, you are probably right. I will change it.

Reply:
Done.